src/components/Pages/Submit/Form/store/types.ts

```diff
@@ -0,0 +1,39 @@
+export type Article = {
```

Comment:
Typescript for me mostly just helps me keep track of my store shapes.

JS is so finicky that I can't function without it checking to make sure my hashes should have the given attribute. Plus the typeahead is a nice bonus.